Tasks/VsTest/helpers.ts

```diff
@@ -22,6 +22,7 @@ export class Constants {
 export class Helper {
     public static addToProcessEnvVars(envVars: { [key: string]: string; }, name: string, value: string) {
         if (!this.isNullEmptyOrUndefined(value)) {
+            tl.debug('Setting the process env var :' + name + ' to :' + value);
```

Comment:
This how it prints DTA access token. So safe to go..
 ##[debug]Setting the process env var :DTA.AccessToken to :********

Reply:
@RanjanarMS Still avoid printing access token info in the log. The build is clever enough to mask